conf/ocsci/collect_memory_usage.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Mayne worth to add some comment what this config file does

Reply:
done